bin/util/bibblio_export

```diff
@@ -25,12 +26,24 @@ parser.add_argument(
     '--languages', type=unicode, nargs='*', default=u'eng',
     help='Look for Works in these languages'
 )
+parser.add_argument(
+    '--fiction', action='store_true',
+    help='Export fiction and nonfiction texts'
+)
```

Comment:
I think keeping fiction and nonfiction separate is appropriate. If that's what you're doing then the `help` should be changed to "Export fiction texts instead of nonfiction".